frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -170,7 +174,7 @@ export default class FirebaseInteractor {
     let wordIds = words.docs.map((word) => word.id);
     let newAssessmentFields = {
       currentIndex: 0,
-      durationInSeconds: 0,
+      durationsInSeconds: 0,
```

Comment:
was this named incorrectly or did you just end up changing the field name to be more accurate?

Reply:
in some places it was named durationInSeconds and some it was named durationsInSeconds, and some database entries actually had both, but durationInSeconds was always 0 so I just decided to use the durationsInSeconds even though grammatically speaking it makes me annoyed